packages/table/src/tableBody.tsx

```diff
@@ -100,14 +105,18 @@ export class TableBody extends React.Component<ITableBodyProps, {}> {
     }
 
     public render() {
-        const { grid, rowIndexStart, rowIndexEnd, columnIndexStart, columnIndexEnd } = this.props;
+        const { getColumnProps, grid, rowIndexStart, rowIndexEnd, columnIndexStart, columnIndexEnd } = this.props;
         const cells: Array<React.ReactElement<any>> = [];
-        for (let rowIndex = rowIndexStart; rowIndex <= rowIndexEnd; rowIndex++) {
-            for (let columnIndex = columnIndexStart; columnIndex <= columnIndexEnd; columnIndex++) {
+        for (let columnIndex = columnIndexStart; columnIndex <= columnIndexEnd; columnIndex++) {
+            const loadingOptions = grid.isGhostIndex(0, columnIndex)
```

Comment:
This check means that ghost cells will never have valid loadingOptions, therefore isLoading will always be false, therefore the attempts to render the skeleton in `renderGhostCell` will never happen.